src/main/cpp/t01_min.cpp

```diff
@@ -22,10 +22,17 @@
 using namespace std;
 
 //function min 
-
+int min(int a, int b){
+    if(a < b) return a;
+    else return b;
+}
 //function min4 
-
+int min4(int a,int b,int c,int d){
+    min(min(a, b), min(c, d));
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Return value of function min() is not used.](https://app.codacy.com/app/thejerome/IFMO_CPP_programming_20190915/pullRequest?prid=4593627)